sorting/bubble_sort_2.c

```diff
@@ -1,46 +1,84 @@
+/**
+ * @file
+ * @brief implementation of [Bubble sort](https://en.wikipedia.org/wiki/Bubble_sort) algorithm
+ * @details
+ * worst-case: O(n^2)
+ * best-case: O(n)
+ * average-complexity: O(n^2)
+ */
+
 #include <stdio.h>
 #include <stdlib.h>
+#include <assert.h>
+#include <stdbool.h>
```

Comment:
Please add a brief description of what the library adds (see the example below).
```c++
#include <iostream>   /// for IO operations
#include <vector>    /// for std::vector
```

Reply:
Oops, actually forgot to remove `stdio.h`. Thanks for the tip 👍🏻